event_tracer.gemspec

```diff
@@ -19,8 +19,10 @@ Gem::Specification.new do |spec|
   spec.require_paths = %w[lib/event_tracer lib]
 
   spec.add_runtime_dependency 'concurrent-ruby'
+  spec.add_runtime_dependency 'dry-configurable'
 
   spec.add_development_dependency "bundler", ">= 2.1.4"
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "rspec", "~> 3.0"
+  spec.add_development_dependency "timecop"
```

Comment:
We add `add_development_dependency` when the gem requires it to work in development environment of the main app. If we only require it when developing the gem itself, we can put it in `Gemfile` instead.